### PR TITLE
move files to filters and transformers-1

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5038

## Description
Move AfterRelativeDate, CheckIfSubdomain, CIDRBiggerThanPrefix, GreaterCidrNumAddresses and InRange form 
CommonScripts to FiltersAndTransformers.

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [X] Yes
       - Further details:
   - [ ] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Comment by sapirshuker** [comment]
All validation failed on 
[BA114] - Pack folder names cannot be changed, please rename it back to CommonScripts. If you wish to rename the pack, you can edit the name field in pack_metadata.json, and the pack will be shown in the Marketplace accordingly.
If the file wasn't renamed, try pulling changes from master and re-run validations

File: Packs/CommonScripts/ReleaseNotes/1_10_35.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"Moved AfterRelativeDate, CheckIfSubdomain, CIDRBiggerThanPrefix, GreaterCidrNumAddresses, InRange scripts to *FiltersAndTransformers* pack"}
```
Comment (Shellyber) [inline Packs/CommonScripts/ReleaseNotes/1_10_35.json:1]:
```suggestion
{"breakingChanges":true,"breakingChangesNotes":"The scripts: AfterRelativeDate, CheckIfSubdomain, CIDRBiggerThanPrefix, GreaterCidrNumAddresses, InRange moved to *FiltersAndTransformers* pack". Make sure to update the *FiltersAndTransformers* pack to the latest version in order to use the scripts} 
```

**Review by Shellyber — COMMENTED** [review]
Let's consult with @ShirleyDenkberg  regarding the release notes for both packs and the breaking changes note

**Review by Shellyber — APPROVED** [review]

**Comment by sapirshuker** [comment]
Validation failed only on BA114 error

**Pull request merged**